.github/workflows/wheel-builder.yml

```diff
@@ -123,25 +127,37 @@ jobs:
         run: mv .github/stdint-nonsense/stdint.h C:/libsodium/include/stdint.h
         if: matrix.PYTHON.VERSION == '2.7'
       - name: Install wheel and our Python dependencies
-        run: pip install wheel cffi six
-
+        run: python -m pip install -U pip wheel cffi six
+      - name: Download the sdist
+        run: pip download pynacl==${{ github.event.inputs.version }} --no-deps --no-binary pynacl && tar zxvf PyNaCl* && mkdir wheelhouse
+        shell: bash
       - name: Build the wheel (py2)
-        run: pip wheel pynacl==${{ github.event.client_payload.BUILD_VERSION }} --wheel-dir=wheelhouse --no-binary pynacl
+        run: cd PyNaCl* && python setup.py bdist_wheel && mv dist/PyNaCl*.whl ../wheelhouse
         env:
           PYNACL_SODIUM_LIBRARY_NAME: sodium
           PYNACL_SODIUM_STATIC: 1
           SODIUM_INSTALL: system
         if: matrix.PYTHON.VERSION == '2.7'
-      - name: Build the wheel (py3)
+      - name: Build the wheel (py3 no abi3)
+        run: |
+          call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Common7\Tools\VsDevCmd.bat" -no_logo -arch=${{ matrix.WINDOWS.VS_ARCH }}
+          cd PyNaCl* && python setup.py bdist_wheel && mv dist/PyNaCl*.whl ../wheelhouse
+        shell: cmd
+        env:
+          PYNACL_SODIUM_LIBRARY_NAME: sodium
+          PYNACL_SODIUM_STATIC: 1
+          SODIUM_INSTALL: system
+        if: matrix.PYTHON.VERSION != '2.7' && matrix.PYTHON.USE_ABI3 != 'true'
+      - name: Build the wheel (py3 abi3)
         run: |
           call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Common7\Tools\VsDevCmd.bat" -no_logo -arch=${{ matrix.WINDOWS.VS_ARCH }}
-          pip wheel pynacl==${{ github.event.client_payload.BUILD_VERSION }} --wheel-dir=wheelhouse --no-binary pynacl
+          cd PyNaCl* && python setup.py bdist_wheel  --py-limited-api=${{ matrix.PYTHON.ABI_VERSION }} && mv dist/PyNaCl*.whl ../wheelhouse
```

Comment:
is this valid cmd syntax?

Reply:
Yeah we do this in cryptography without issue.